### ISSUE TITLE
Main image not updated on product website image after selecting other image of the carousel

### ISSUE THREAD

**Issue by antoniocanovas** [OdooCommunityWidgets/website_multi_image#32]
I am working in website with products that include several variants and images.
Each attribute has a different main image.
I can change the attribute, and the main image of this attribute is shown, but when I choose another image of the product in the carousel this is frozen and don't change when I select other variant.
Good job!

**Comment by lukebranch** [comment]
@konerycloud ,

Thanks for your interest in this module. I have not yet built these features into the module, however it is on the roadmap. I have an upcoming project that will require attribute swatches with images (ie. handled like Magento 1.9+) and separate sets of product images per variant which I believe it what you're looking for, however due to time constraints and other projects it's not going to happen anytime this month.

I will be rebuilding the module from scratch to base it on the product_multi_image module in the OCA/product-attribute repo, however this module is still being reviewed. Once the product_multi_image module is merged into the 8.0 branch of OCA/product-attribute.

In the meantime, please consider providing me with an animated GIF using:

http://www.cockos.com/licecap/

or similar so that I can see the behaviour you've described and can try and reproduce it on my end and merge a workaround for it in the short term.
